README.md

```diff
@@ -613,6 +613,20 @@ In src/sensor/settings.py, the OAuth `USER_NAME` and `PASSWORD` are set to be th
 as `CLIENT_ID` and `CLIENT_SECRET`. This may need to change depending on your
 authorization server.
 
+### Data File Encryption
+
+The data file is encrypted on disk by default using cryptography Fernet module. Note
```

Comment:
I will update the README to clarify this. According to the cryptography documentation, the limit on file size is that the file must fit entirely in memory. So it is not unavoidable by not using tmpfs. I decided to use tmpfs to prevent writing unecrypted data to disk when decrypting/creating SigMF archive since deleting files doesn't usually remove data from the disk.

Reply:
That makes sense- I'll leave this comment unresolved pending the README updates to clarify. Thanks!